showcase/app/components/shw/text/index.gts

```diff
@@ -4,8 +4,9 @@
  */
 
 import Component from '@glimmer/component';
-import { action } from '@ember/object';
 import { assert } from '@ember/debug';
+import { eq } from 'ember-truth-helpers';
```

Comment:
[thought] nice! didn't realize modifiers should be imported this way (makes sense, now that I see it)